aait/backend/group-2A/Persistence/Repositories/LikeRepository.cs

```diff
@@ -0,0 +1,42 @@
+using Application.Contracts.Persistance;
+using Application.DTO.UserDTO;
+using Domain.Entities;
+using Microsoft.EntityFrameworkCore;
+
+namespace Infrastructure.Persistance.Repository;
+
+public class LikeRepository : ILikeRepository{
+    private readonly SocialSyncDbContext _dbContext;
+    private readonly DbSet<Like> _dbLikes;
+
+    public LikeRepository(SocialSyncDbContext context){
+        _dbContext = context;
+        _dbLikes = context.Set<Like>();
```

Comment:
instead setting dbContext for like whenever likeRepository object is created , you can define the dbSet for like in the dbcontext. define dbSets for all of the tables in the db context.